repository-data/webfiles/src/main/resources/site/freemarker/common/apicatalogue/scrollable-filter-nav.ftl

```diff
@@ -82,32 +84,40 @@
         </#if>
         <div><#-- This div is needed to add vertical spacing between checkboxes -->
         <span class="nhsd-a-checkbox">
-            <label>
+            <label class="filter-label <#if filter.description()??>filter-label__described</#if>">
             <#if filter.selectable>
                 <a title="Filter by ${filter.displayName}"
                    href="<@renderUrl baseUrl=baseUrl showRetired=showRetired filters=filtersParam />"
                    class="nhsd-a-checkbox__label nhsd-t-body-s <#if filter.selected>selected</#if> <#if filter.entries?has_content>nhsd-!t-font-weight-bold</#if>">
-                    <input type="checkbox">&nbsp;${filter.displayName}
+                    <input type="checkbox">
+                    <span class="<#if !responsive>filter-label__text</#if>">${filter.displayName}</span>
                 </a>
                 <span class="checkmark <#if filter.selected>selected</#if>"></span>
             <#else>
                 <a class="nhsd-a-checkbox__label nhsd-t-body-s nhsd-api-catalogue__filter-unavailable <#if filter.entries?has_content>nhsd-!t-font-weight-bold</#if>">
-                    <input type="checkbox" disabled>&nbsp;${filter.displayName}
+                    <input type="checkbox" disabled>
+                    <span class="<#if !responsive>filter-label__text</#if>">${filter.displayName}</span>
                 </a>
                 <span class="checkmark <#if filter.selected>selected</#if>"></span>
             </#if>
-                <p class="nhsd-a-checkbox__hint"></p>
             </label>
+            <@filterDescription responsive filter.description()/>
```

Comment:
since the filterDescription is rendered _outside_ of the \<a\> element, am i right thinking it isn't actually "clickable"? in this case it might be better for the cursor to not be a pointer when hovering over it